daemon/call.c

```diff
@@ -206,6 +213,16 @@ static void call_timer_iterator(struct call *c, struct iterator_helper *hlp) {
 
 		/* valid stream */
 
+		// ignore RTCP Streams
+		if (PS_ISSET(ps, RTCP))
```

Comment:
I think it might be more appropriate to use `if (!PS_ISSET(ps, RTP))` here? (Streams can be both RTP and RTCP in case of RTCP-mux)

Reply:
Yes, this sounds better. I hadn't RTCP-mux in mind, because it's not used in our case.